### PR TITLE
Basic setup

### PR DESCRIPTION
**Pull request by NoelLincoln**
Feat:
Create index.html and add h1 tag inside header tag
Create style.css and add background color to header tag using css class selector
Create linters.yml file to check for errors in the code
Add node_modules directory to .gitignore
Create test.md file and add it to .gitignore

### CODE REVIEW

**Review by ademibowale — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @DevaWinner , 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight 🥇

✅ Descriptive Pull Request title and summary. ✔️
✅ Good use of Github flow ✔️
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.

### Changes Required

- [ ]  _Please check the comments under the review._
- [ ] Good job using the Microverse template for your README. However, I noticed that some sections are missing from your content table. Please ensure that you include all the main sections in your README as it is a requirement of the project to follow the [Microverse template](https://github.com/microverseinc/readme-template/blob/master/README.md). kindly modify it to fit your custome,The only section that is optional to include is the FAQ section. Also, if your project doesn't require deployment, you can remove the Live Demo section. 🎯
If you don't have a logo you can remove it too.

- [ ] The only optional section is FAQ.
You should delete the live demo section if the you decides not to deploy the application. If project requirements say students should deploy, they should put the link.
Make sure the you have added the section Future Features, including 1-3 potential features to be added.


you should remove the section description, i.e. "> Describe the tech stack and include only the relevant sections that apply to your project." should be removed and filled in with your custom content.
remove template logo. Optionally, the students can replace with their own logo if they have one.you submit for the new review.

Kindly follow the guildline above to modify your readme step by step before you submit for the new review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by NoelLincoln** [comment]
After including the main sections, can I leave some sub-sections blank or do I remove them because it does not relate to my project? For example, under getting started where there's no setup needed, no install, no usage etc.

**Comment by ademibowale** [comment]
> After including the main sections, can I leave some sub-sections blank or do I remove them because it does not relate to my project? For example, under getting started where there's no setup needed, no install, no usage etc.



> After including the main sections, can I leave some sub-sections blank or do I remove them because it does not relate to my project? For example, under getting started where there's no setup needed, no install, no usage etc.


On narate how your project can be clone with yoir repo link ,and the usage , only section that is optional to include is the FAQ section.

**Comment by NoelLincoln** [comment]
Okay

**Comment by NoelLincoln** [comment]
[@Adebowale](https://github.com/Adebowale) 
Should I close this pull request and do a new one after I'm done with the corrections?

File: README.md [hunk 1]
```diff
@@ -41,6 +143,5 @@ Hello world program
 
 This project is [MIT](./LICENSE) licensed.
 
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
-
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
+```
```
Comment (roshan-bajgain) [inline README.md:147]: - [ ] I notice that your MIT license is broken please fix this license so the README file looks professional. Here is the [LInk](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) how you can create license in your README file.

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @NoelLincoln, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (NoelLincoln) [reply README.md:147]: @roshan-bajgain 
I've fixed the broken link and used the MIT license.
Thanks

**Comment by NoelLincoln** [comment]
I have fixed the broken MIT license from the README

**Review by Alusp — APPROVED** [review]
Hi <STUDENT_MENTION>, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Congrats

## Highlights

- [ ] Descriptive Pull Request title and summary :white_check_mark: 
- [ ] Good use of Github flow. :white_check_mark: 
- [ ] All linters are checked. :white_check_mark: 
- [ ] Project is professionally documented. :white_check_mark: 
- [ ] The right GitHub flow method has been used. :white_check_mark: 
- [ ] Professional pull request title and description. :white_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by NoelLincoln** [comment]
Thank you @Alusp

**Pull request merged**